signal_scanner_bot/env.py

```diff
@@ -13,8 +14,43 @@
 class _State:
     """Class for holding global state across threads/tasks"""
 
-    LISTENING = False
-    STOP_REQUESTED = False
+    # Initialize state, set LISTENING to true if state file exists,
+    # sets false if not.
+    def __init__(self, file: str):
+        self.file = file
+        self.LISTENING = True if os.path.isfile(file) else False
+        self.STOP_REQUESTED = False
```

Comment:
If we pass `file` in as a `pathlib.Path` object, we can change this to the following:
```suggestion
    def __init__(self, file: Path):
        self.file = file
        self.LISTENING = self.file.exists()
        self.STOP_REQUESTED = False
```